setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-#
+# This file is part of Invenio.
+# Copyright (C) 2015-2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('CHANGES.rst') as history_file:
+    history = history_file.read()
+
+requirements = ['Click>=6.0', ]
```

Comment:
Align with https://github.com/inveniosoftware/invenio-search/blob/cb4edcf7d56b60c26618e9da64e397653c326071/setup.py#L18-L83

In `extras_requires` include `docs` and `tests`.